diff -r 12bbb2eed623 src/util_flipflop.js
--- a/src/util_flipflop.js	Thu Nov 21 10:19:47 2013 +0900
+++ b/src/util_flipflop.js	Thu Nov 21 10:20:29 2013 +0900
@@ -65,7 +65,9 @@
         ["<sub>", "</sub>", "subscript", "@vertical-align", ["sub", "sub","baseline"], true],
         ["<sc>", "</sc>", "smallcaps", "@font-variant", ["small-caps", "small-caps","normal"], true],
         ["<span style=\"font-variant:small-caps;\">", "</span>", "smallcaps", "@font-variant", ["small-caps", "normal","normal"], true],
+        ["{{", "}}", "passthrough", "@passthrough", ["true", "true","true"], true],
         ["<span class=\"nocase\">", "</span>", "passthrough", "@passthrough", ["true", "true","true"], true],
+        ["{", "}", "passthrough", "@passthrough", ["true", "true","true"], true],
         ["<span class=\"nodecor\">", "</span>", "passthrough", "@passthrough", ["true", "true","true"], true],
         ['"',  '"',  "quotes",  "@quotes",  ["true",  "inner","true"],  "'"],
         [" '",  "'",  "quotes",  "@quotes",  ["inner",  "true","true"],  '"']
@@ -413,7 +415,7 @@
                 //
                 // CSL.debug(this.okReverseTagsHash[this.blob.alldecor[0][0].join("-is-")]);
                 //
-                if (tag === "<span class=\"nodecor\">") {
+                if (tag === "<span class=\"nodecor\">" || tag === "{{") {
                     fulldecor = this.state[this.state.tmp.area].opt.topdecor.concat(this.blob.alldecor).concat([[["@quotes", "inner"]]]);
 
                     lenB = fulldecor.length;
